daemon/kernel_modifications.md

```diff
@@ -12,7 +12,7 @@ but feel free to browse using apt-cache search linux-image-extra to find all pos
 install. You'll want to install the latest.
 
 ``` bash
-apt-get install linux-image-4.4.0-72-generic linux-image-extra-4.4.0-72-generic
+apt-get install linux-image-4.4.0-131-generic linux-image-extra-4.4.0-131-generic
```

Comment:
There is a meta package `linux-image-generic` available that points to the latest version. Not sure if `linux-image-extra-generic` exists as well, if so we should probably use it.